report-viewer/src/model/Match.ts

```diff
@@ -7,15 +7,15 @@
  * @property startInSecond - Starting line of the match in the second file.
  * @property endInSecond - Ending line of the match in the second file.
  * @property tokens - Number of tokens in the match.
- * @property color - Color of the match.
+ * @property colorIndex - Index of the color to use for the match.
  */
-export type Match = {
+export interface Match {
   firstFile: string
   secondFile: string
   startInFirst: number
   endInFirst: number
   startInSecond: number
   endInSecond: number
   tokens: number
-  color: string
+  colorIndex?: number
```

Comment:
This is an optional, since for the color comparison we need a number here that is no possible index. Another option would be putting something like -1 in the initialization here. To not cause an error I made `getMatchColor` return a fully transparent color on an undefined index.

We only save the index here, so we can set a different alpha for different occasions, for example for dark and light mode if it is needed